ArcGISRuntimeSDKQt_SampleViewers/samples.pro

```diff
@@ -5,5 +5,4 @@ mac {
 TEMPLATE = subdirs
 
 SUBDIRS += \
-    $$PWD/ArcGISRuntimeSDKQt_QMLSamples \
```

Comment:
This is a good point. I considered the building of the sample apps but not the archiving that happens afterwards. I'll double check and see before proceeding with this. Otherwise Ill make the necessary changes so that QML continues to build and we can delay this specific work until DevOps is ready.

Reply:
Discussed with DevOps looks like all of the archiving uses wildcards from a parent output folder. We should be safe to do this.